corehq/toggles/__init__.py

```diff
@@ -2569,6 +2569,21 @@ def _commtrackify(domain_name, toggle_is_enabled):
     namespaces=[NAMESPACE_DOMAIN],
 )
 
+ENTERPRISE_DASHBOARD_IMPROVEMENTS = StaticToggle(
+    'enterprise_dashboard_improvements',
+    'Shows an improved version of enterprise dashboard during development',
+    TAG_PRODUCT,
+    namespaces=[NAMESPACE_USER]
+)
```

Comment:
Did this guy sneak in here?

Reply:
Oooh must've been a bad merge. Good catch, thank you.